ldaptor/protocols/pureber.py

```diff
@@ -362,7 +361,11 @@ def inherit(self):
     def __repr__(self):
         identities = []
         for tag, class_ in self.Identities.items():
-            identities.append('0x%02x: %s' % (tag, class_.__name__))
+            identities.append('0x{:02x}: {}'.format(tag, class_.__name__))
+
+        if sys.version_info < (3, 6):
```

Comment:
twisted is [about to drop support for python 3.5](https://github.com/twisted/twisted/pull/1386) as it's eol